packages/gcf-utils/package.json

```diff
@@ -70,7 +70,7 @@
     "snap-shot-it": "^7.9.6",
     "sonic-boom": "^3.2.0",
     "stream-mock": "^2.0.5",
-    "typescript": "~4.8.2"
+    "typescript": "4.8.2"
```

Comment:
4.8.4 was released 3 months ago. Maybe we can use it instead?

Reply:
Done.